### PR TITLE
Php - Version to merge - No Cake (but ready as side provisionner)

### PR DESCRIPTION
**Pull request by mtancoigne**
So this is the simple PHP box with no Zeus modifications, without CakePHP (I kept a second provision.sh file for a later integration)

### CODE REVIEW

**Comment by byteknacker** [comment]
Also I would like to propose that any change in the provisioner by the user as an upgrade or modification should be programmatically designed as commands in zeus main script or libraries. Instead of the user directly writing anything in the provision.sh, the user will just to have to write it as plain text or enter them as arguments and they will be picked up as regex from the script and smartly assigned the correct output text to be added at the right place in the provision.sh which then will be used.

**Comment by alayek** [comment]
Maybe we can use YAML. It's smarter than text files, and yet simpler.

**Comment by byteknacker** [comment]
Yes any text file format would be good to be honest. The user can just write down a comma separated list of fuzzy names of packages to be installed and zeus will figure it out.

**Comment by byteknacker** [comment]
Just started reviewing :) <br><br>:octocat: _Sent from [GH](http://nodegh.io)._

File: php/README.md [hunk 1]
```diff
@@ -1,53 +1,92 @@
-# Work in progress
+# Zeus for PHP
```
Comment (atjonathan) [inline php/README.md]: ~~Zeus for PHP~~ Instructions on how to use Zeus for PHP

File: php/README.md [hunk 2]
```diff
@@ -1,53 +1,92 @@
...
 
-## Prerequisites:
-  - VirtualBox
-  - Vagrant (with utils module)
-  - An Heroku account
+## Filesystem
+ The document root of the Apache sercer is located in the `/vagrant/www` directory on the _guest box_, wich is your base of work for your PHP files.
```
Comment (ghost) [inline php/README.md:4]: ~~sercer~~ server

**Comment by byteknacker** [comment]
LGTM! :thumbsup:

**Pull request merged**